src/zac/core/views/processes.py

```diff
@@ -117,7 +117,34 @@ def form_valid(self, form: ClaimTaskForm):
         raise SuspiciousOperation("Unsafe HTTP_REFERER detected")
 
 
-class PerformTaskView(PermissionRequiredMixin, FormView):
+class FormSetMixin:
+    def get_formset_class(self):
+        task = self._get_task()
+        return task.form.get("formset")
+
+    def get_formset(self, **kwargs):
+        formset_class = self.get_formset_class()
+        if not formset_class:
+            return
+
+        # FIXME how to feed initial data into formset ???
+        # kwargs = {"initial": formset_class.get_initial()}
+
+        if self.request.method in ("POST", "PUT"):
```

Comment:
not sure why PUT is in here
```suggestion
        if self.request.method == "POST":
```